core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -904,7 +904,12 @@ private[spark] class SparkSubmit extends Logging {
       childArgs ++= Seq("--verbose")
     }
 
-    sparkConf.set("spark.app.submitTime", System.currentTimeMillis().toString)
+    val setSubmitTimeInClusterModeDriver =
+      sparkConf.getBoolean("spark.kubernetes.setSubmitTimeInDriver", false)
```

Comment:
When we introduce a new feature, we hide it first at least for a single Apache Spark release in order to provide a testing capability without the default behavior changes. In other words, this should be `true`.
```scala
- sparkConf.getBoolean("spark.kubernetes.setSubmitTimeInDriver", false)
- sparkConf.getBoolean("spark.kubernetes.setSubmitTimeInDriver", true)
```

Reply:
@dongjoon-hyun OK.